t/core/etcd-mtls.t

```diff
@@ -82,12 +82,8 @@ deployment:
     }
 --- request
 GET /t
---- grep_error_log eval
-qr/init_by_lua:\d+: \S+/
---- grep_error_log_out
-init_by_lua:12: ab
```

Comment:
why did you remove these?

Reply:
@shreemaan-abhishek 

As you can see on line 88, some smarty-pants developer even asserted in the log that the line numbers were fixed, and that it was impossible to init block unchanged, so the line numbers would change.

It is now modified to use multiple qr blocks at the same time with "and" conditions for log checking.